Uniletter/Sources/MyPage/MyPageViewModel.swift

```diff
@@ -42,19 +42,17 @@ enum SectionType: CaseIterable {
     }
 }
 
-struct UserInfo {
-    var nickname: String?
-    var image: UIImage?
-}
-
 class MyPageViewModel {
     
     static let shared = MyPageViewModel()
```

Comment:
싱글톤으로 만드시려고 하는 것 같아요! 싱글톤으로 만드실 때에는
initializer를 private로 선언해주어야 다른 곳에서 `MyPageViewModel`을 생성해서 덮어쓰는일을 방지할 수 있어요!!

```suggestion
    static let shared = MyPageViewModel()
    
    private func init() {}
```

그리고 이건 단순히 궁금한건데 ViewModel을 싱글톤으로 하신 이유가 있으실까요? 👀

Reply:
엇 리뷰감사합니다. MyPageViewModel에 있는 userName이랑 userImage를 마이페이지화면과 프로필수정화면에서 사용하기 위해 싱글톤으로 선언해줬습니다.

Reply:
킹명섭 피드백에 그저 감사할 따름입니다! 감사합니다!